### ISSUE TITLE
The REPL help text for 'doc' is incorrect

### ISSUE THREAD

**Issue by oubiwann** [lfe/lfe#401]
The help text for the built-in `doc` functions is incorrect. It states:

```
(doc mod)      -- documentation of a module
(doc mod mac)  -- documentation of a macro
(doc m f a)    -- documentation of a function
```

However, the actual arguments should be provided as follows:

```
(doc mod)
(doc mod:macro
(doc mod:fun/arity)
```

**Comment by rvirding** [comment]
Fixed in b139867cc8d0e580bbf5643c31e38e48b7cc2105

**Comment by oubiwann** [comment]
Thanks, @rvirding! I'll remove the warning about this from the LFE Chineual 😄
